lib/reportportal.rb

```diff
@@ -74,27 +76,22 @@ def send_log(status, message, time)
       end
     end
 
-    def send_file(status, path, label = nil, time = now, mime_type = 'image/png')
-      unless File.file?(path)
+    def send_file(status, path_or_src, label = nil, time = now, mime_type = 'application/octet-stream')
```

Comment:
why did you change default mime type?

Reply:
Default mime type is used when a file is passed to the method via source.  In these cases, we don't know which mime/type the file has.
Previously it was assumed that file type is PNG but it could be anything.

Reply:
It may occur errors if someone is skipping PNG as default type.

Reply:
Ok, maybe you are right given that Report Portal has a built-in support only for images.